apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -41,41 +36,23 @@ class Courses extends Component {
       buttonText: i18n.coursesLearnHeroButton(),
     };
 
-    // Apply overrides if this is the "Teach" view and log teacher visiting this page.
-    if (isTeacher) {
-      heroStrings = {
-        headingText: i18n.coursesTeachHeroHeading(),
-        subHeadingText: i18n.coursesTeachHeroSubHeading(),
-        buttonText: i18n.coursesTeachHeroButton(),
-      };
-    }
-
     // We show a long version of the banner when you're signed out,
     // so add a description string.
     if (isSignedOut) {
-      heroStrings.description = isTeacher
-        ? i18n.coursesTeachHeroDescription()
-        : i18n.coursesLearnHeroDescription();
+      heroStrings.description = i18n.coursesLearnHeroDescription();
     }
     return heroStrings;
   }
 
   render() {
-    const {
-      isEnglish,
-      isTeacher,
-      isSignedOut,
-      modernElementaryCoursesAvailable,
-      specialAnnouncement,
-    } = this.props;
+    const {isEnglish, isSignedOut, modernElementaryCoursesAvailable} =
+      this.props;
 
     const {headingText, subHeadingText, description, buttonText} =
       this.getHeroStrings();
 
     // Verify background image works for both LTR and RTL languages.
-    const backgroundUrl = isTeacher
-      ? '/shared/images/banners/courses-hero-teacher.jpg'
```

Comment:
Is the banner image from the teach page used anywhere else? should we remove it?

Reply:
Yep you're right it's not used anywhere else! Removed it!